FEXCore/Source/Interface/IR/IRDumper.cpp

```diff
@@ -183,6 +183,15 @@ static void PrintArg(fextl::stringstream *out, [[maybe_unused]] IRListView const
   }
 }
 
+static void PrintArg(fextl::stringstream *out, [[maybe_unused]] IRListView const* IR, FEXCore::IR::NamedVectorConstant Arg) {
+  switch (Arg) {
+    case FEXCore::IR::NamedVectorConstant::NAMED_VECTOR_INCREMENTAL_U16_INDEX: *out << "u16_incremental_index"; break;
```

Comment:
Considering how long these identifiers are, maybe line break the body onto a new line

Reply:
Split between lines.